common/tabs_interference_carousel/reports/pre-a11y-1.json

```diff
@@ -0,0 +1,444 @@
+{
+  "@context": "https://wet-boew.github.io/vocab/context/2023/conformance-report-en.json",
+
+  "@type": [ "earl:Assertion", "acr:AssessmentReport" ],
+
+  "earl:subject": {
+    "@id": "_:subject",
+    "dct:references": "https://wet-boew.github.io/GCWeb/sites/language/language-en.html",
+    "@type": [
+      "earl:TestSubject",
+      "schema:WebPage"
+    ],
+    "dct:description": "Evaluation of how the Tabs Interface - Carousel component meets accessibility requirements",
+    "earl:pointer": {
+      "@type": "oa:CssSelector",
+      "@value": ""
```

Comment:
Missing the CSS selector.

Please make it also specific to the carrousel and its variants